pkg/spdk/engine.go

```diff
@@ -938,6 +938,10 @@ func (e *Engine) ReplicaShallowCopy(dstReplicaName, dstReplicaAddress string) (e
 	defer func() {
 		// Blindly mark the rebuilding replica as mode ERR now.
 		if err != nil {
+			// Blindly send rebuilding finish for src replica.
+			if srcReplicaErr := srcReplicaServiceCli.ReplicaRebuildingSrcFinish(srcReplicaName, dstReplicaName); srcReplicaErr != nil {
```

Comment:
Do you want to wrap `srcReplicaErr` as well? or it's meaningless here?

Reply:
Typo below. Will just print out it.